### PR TITLE
FormulaWidgetUI component migrated from makeStyles to styled-components + cleanup 

### PR DESCRIPTION
**Pull request by VictorVelarde**
Adapt to styled api + Fix in the behaviour of prefix / suffix

### CODE REVIEW

File: packages/react-ui/src/widgets/FormulaWidgetUI.js [hunk 1]
```diff
@@ -72,26 +67,25 @@ function FormulaWidgetUI(props) {
   const formattedValue = formatter(value);
 
   return (
-    <Box className={classes.root}>
+    <Root>
```
Comment (vmilan) [inline packages/react-ui/src/widgets/FormulaWidgetUI.js:70]: I think if you replace this `Root` component with our `Typography`, you don't need this custom styled component:

`<Typography variant='h5' component='div' weight='medium'>`

Comment (VictorVelarde) [reply packages/react-ui/src/widgets/FormulaWidgetUI.js:70]: Ok, let me check

Comment (VictorVelarde) [reply packages/react-ui/src/widgets/FormulaWidgetUI.js]: image
Change done

**Comment by VictorVelarde** [comment]
Changes done, if you're ok, we'll merge it tomorrow @vmilan

**Review by vmilan — APPROVED** [review]
👍

**Comment by vmilan** [comment]
@VictorVelarde all looking good, but I've seen the build is failing because doesn't found `carto react package`
Screenshot 2023-05-09 at 09 55 06

I've done a change in the import: https://github.com/CartoDB/carto-react/pull/666/commits/cbcab4a047ca2688a49d85afcad28ab59c0cf96c

Let's see if this fix the issue.

**Comment by vmilan** [comment]
@VictorVelarde all green now, if you are ok we can land it

**Pull request merged**